code-editor/auto-stop-idle/on-start.sh

```diff
@@ -37,4 +54,4 @@ sudo /bin/bash -c "echo 'AWS_CONTAINER_CREDENTIALS_RELATIVE_URI=$AWS_CONTAINER_C
 
 # Add script to crontab for root.
 echo "Adding autostop idle Python script to crontab..."
-echo "*/2 * * * * /bin/bash -ic '$CONDA_HOME/python $PYTHON_SCRIPT_PATH --time $IDLE_TIME_IN_SECONDS --region $AWS_DEFAULT_REGION >> $LOG_FILE'" | sudo crontab -
+echo "*/2 * * * * /bin/bash -ic '$CONDA_HOME/python $PYTHON_SCRIPT_PATH --time $IDLE_TIME_IN_SECONDS --region $AWS_DEFAULT_REGION >> $LOG_FILE'" | sudo crontab -
```

Comment:
Is this line actually changed?

Reply:
Were lines 40-56 removed?

Reply:
@edwardps no the below line was not changed. Line number 40 was moved to 57. 

`echo "*/2 * * * * /bin/bash -ic '$CONDA_HOME/python $PYTHON_SCRIPT_PATH --time $IDLE_TIME_IN_SECONDS --region $AWS_DEFAULT_REGION >> $LOG_FILE'" | sudo crontab -
`

@aws-spenceng nope. No lines were removed. We modified the logic in the line [20-21 ](https://github.com/aws-samples/sagemaker-studio-apps-lifecycle-config-examples/blob/9c305c0ef6fb2b71c7d9329e193ba719e9db9c1b/code-editor/auto-stop-idle/on-start.sh#L20) and added extra lines [21-38](https://github.com/lavaraja/sagemaker-studio-apps-lifecycle-config-examples/blame/112c43e36633b94d8ec2a0327881ba188e68fe77/code-editor/auto-stop-idle/on-start.sh#L21)